core/common/build.gradle.kts

```diff
@@ -19,6 +19,10 @@ plugins {
     id("nowinandroid.android.hilt")
 }
 
+android {
+    namespace = "com.google.samples.apps.nowinandroid.core.common"
+}
```

Comment:
Nit: the `android` block can be omitted.
```kts
android.namespace = "com.google.samples.apps.nowinandroid.core.common"
```

Reply:
I prefer to always use blocks in case more properties are added in the future, then someone would need to refactor this into a block